llvm/test/CodeGen/NVPTX/load-store-vectors-256.ll

```diff
@@ -1506,3 +1506,98 @@ define void @local_volatile_4xdouble(ptr addrspace(5) %a, ptr addrspace(5) %b) {
   store volatile <4 x double> %a.load, ptr addrspace(5) %b
   ret void
 }
+
+define void @test_i256_global(ptr addrspace(1) %a, ptr addrspace(1) %b) {
```

Comment:
I'd add a test case for non-global AS to make sure 256 bit loads/stores are not used there, and, maybe a test case for some other 256-bit type (e.g. a <2 x 128> or <4 x double>) to have a reference for what's expected to happen (or not).